.gitignore

```diff
@@ -51,9 +51,12 @@ packages
 *.user
 .ntvs_analysis.dat
 obj/*
-.env
+*.env
```

Comment:
I wanted to have sample.env in the repo. It shows customers the environment variables that they need to use. It acts as a template for them. Hence just .env would be good.

Reply:
Done.